### ISSUE TITLE
Live data graph with data gaps

### ISSUE THREAD

**Issue by nightfixed** [jjoe64/GraphView#228]
Hello,
I am workin on an application where I have to display real-time data, on multiple graphs.
On the X-axis, I render timestamps (not the "current" timestamp, but the timestamp of the data rendered on Y-axis)  and on Y-axis there is my data.
The problem I have is that I don't know how to synchronize the data, in case I have gaps in the data. I would need to keep the X-axis scrolling (since time is actually passing) even if I have no values for the Y-axis. So it's like this:

|||||||||||||||||||||||....|||||||||||............|||||||||||||||||||||||||||||||
|| - represents the data
... - represents the gaps in the data

Basically, if I have gaps in the data I still need to somehow display "white" space, in order to scroll the X-axis to account for time. And once I have data again, append the next timestamp to the whitespace that was introduced.  I just don't know how to do this.
I was thinking to somehow create "dummy" data and fill the gaps with that, but it's rather complicated...

Does anyone have a suggestion?

Very much appreciated.
